### ISSUE TITLE
[Feature Request] Custom icon support

### ISSUE THREAD

**Issue by quinnzbellz** [SimpleMobileTools/Simple-Launcher#30]
This would be a really nice feature to have, especially for making a nice minimal setup with a line based icon pack

**Comment by Saijin-Naib** [comment]
Yes, I was interested in using this in conjunction with Arcticons or RetroMode.

**Comment by heschy2** [comment]
It would also be Nice to provide your own SimpleIconPack as part of the Launcher.
You could create a Checkbox in the Settings where you can opt-in to this Icon Pack.
